app/component/scorecard/scorecard.html

```diff
@@ -0,0 +1,752 @@
+        <label for="shot7"></label>
+        <span class="form-control shot-score" id="shot7">{{scorecardCtrl.scorecard.shots[2][6].score}}</span>
+        <label for="shot8"></label>
+        <span class="form-control shot-score" id="shot8">{{scorecardCtrl.scorecard.shots[2][7].score}}</span>
+        <label for="shot9"></label>
+        <span class="form-control shot-score" id="shot9">{{scorecardCtrl.scorecard.shots[2][8].score}}</span>
+        <label for="shot10"></label>
+        <span class="form-control shot-score" id="shot10">{{scorecardCtrl.scorecard.shots[2][9].score}}</span>
+        <label for="shot11"></label>
+        <span class="form-control shot-score" id="shot11">{{scorecardCtrl.scorecard.shots[2][10].score}}</span>
+        <label for="shot12"></label>
+        <span class="form-control shot-score" id="shot12">{{scorecardCtrl.scorecard.shots[2][11].score}}</span>
+        <label for="shot13"></label>
+        <span class="form-control shot-score" id="shot13">{{scorecardCtrl.scorecard.shots[2][12].score}}</span>
+        <label for="shot14"></label>
+        <span class="form-control shot-score" id="shot14">{{scorecardCtrl.scorecard.shots[2][13].score}}</span>
+        <label for="shot15"></label>
+        <span class="form-control shot-score" id="shot15">{{scorecardCtrl.scorecard.shots[2][14].score}}</span>
+        <label for="shot16"></label>
+        <span class="form-control shot-score" id="shot16">{{scorecardCtrl.scorecard.shots[2][15].score}}</span>
+        <label for="shot17"></label>
+        <span class="form-control shot-score" id="shot17">{{scorecardCtrl.scorecard.shots[2][16].score}}</span>
+        <label for="shot18"></label>
+        <span class="form-control shot-score" id="shot18">{{scorecardCtrl.scorecard.shots[2][17].score}}</span>
+        <label for="shot19"></label>
+        <span class="form-control shot-score" id="shot19">{{scorecardCtrl.scorecard.shots[2][18].score}}</span>
+        <label for="shot20"></label>
+        <span class="form-control shot-score" id="shot20">{{scorecardCtrl.scorecard.shots[2][19].score}}</span>
+        <pre class="total-field">{{scorecardCtrl.match3Scores.matchScoreTotal}}</pre>
+      </div>
+    </div>
+    <section  class="scorecard-scoring-totals">
+      <div class="subtotal-score-data">
+        <pre>Match #1 Score:  {{scorecardCtrl.match1Scores.matchScoreTotal}}     X-Count:  {{scorecardCtrl.match1Scores.xCount}}</pre>
+        <pre>Match #2 Score:  {{scorecardCtrl.match2Scores.matchScoreTotal}}     X-Count:  {{scorecardCtrl.match2Scores.xCount}}</pre>
+        <pre>Match #3 Score:  {{scorecardCtrl.match3Scores.matchScoreTotal}}     X-Count:  {{scorecardCtrl.match3Scores.xCount}}</pre>
+      </div>
+      <div class="aggregate-score-data">
+        <pre>Match Aggregate Score: {{scorecardCtrl.totalCompScore}}</pre>
+        <pre>X-Count: {{scorecardCtrl.totalXCount}}</pre>
+      </div>
+    </section>
+  </table>
```

Comment:
This is easily one of the biggest templates ive ever seen,

it would be great to break this out into smaller components that get added to make it more readable
also using more ng-repeats with components cold make this much more readable